frontends/web/src/components/guide/guide.tsx

```diff
@@ -61,14 +62,30 @@ interface ProvidedProps {
 
 type Props = ProvidedProps & SharedProps & TranslateProp;
 
+// guide entries are fetched from the translation files. Manual fallback to is implemented in case an
+// array element is undefined in the current language (can be null if not translated in order, for
```

Comment:
So the idea is that, for example,
```
        "backups": [
            {
                "title": "What is a backup?",
                "text": [ "It is a copy of the seed on an micro SD card. The seed together with your recovery password generates your wallet." ]
            },
            {
                "title": "Can I encrypt the backup?",
```
could be "translated" to German with
```
        "backups": [
            null,
            {
                "title": "Wie kann ich ein Backup verschlüsseln?",
```
if I'm too lazy to also translate the first entry?

Reply:
Correct. More importantly, not due too laziness, but just out of order translation can also produce `null` entries, which crashes the frontend.

Also, if something is missed, it's nicer to show a fallback instead of nothing, like in the rest of the framework like with `t('only.exists.in.english')`, which falls back if not present in another language.

Reply:
> just out of order translation can also produce null entries

How so? Only with explicit `null` entries, or not?

Reply:
No, it is how the translation tool works. If you translate the 3rd entry of a guide, but for some reason have not done the 2nd one yet, the json that comes out will have `null` in the 2nd place. 

Guide array elements are translated as units, if that was the confusion.